clients/go/pkg/audit/interceptor.go

```diff
@@ -138,6 +126,15 @@ func (i *Interceptor) UnaryInterceptor(ctx context.Context, req interface{}, inf
 		Mode:      i.logMode,
 		Timestamp: timestamppb.New(time.Now().UTC()),
 	}
+	// look for justification info and set justification token.
+	if i.getJVSToken(ctx) != "" {
+		if logReq.Context == nil {
+			logReq.Context = &structpb.Struct{
+				Fields: map[string]*structpb.Value{},
+			}
+		}
+		logReq.Context.Fields[justification.TokenHeaderKey] = structpb.NewStringValue(i.getJVSToken(ctx))
+	}
```

Comment:
It seems you can wrap the whole block into a helper func? Also, since the `logReq` is created right above, we can just initialize the `logReq.Context` there.

Reply:
Do we want to  initialize the `logReq.Context`  to an empty struct instead of keeping it nil when there is no jvs token avaialable?

Reply:
helper func added.